src/SampleMods/OWML.EnableDebugMode/EnableDebugMode.cs

```diff
@@ -32,6 +40,7 @@ public void Start()
 		{
 			ModHelper.Console.WriteLine($"In {nameof(EnableDebugMode)}!", MessageType.Info);
 			//ModHelper.HarmonyHelper.EmptyMethod<DebugInputManager>("Awake");
+			ModHelper.HarmonyHelper.AddPrefix<DebugInputManager>("Awake", typeof(ClearPatch), "prefP");
```

Comment:
could you move this and ClearPatch into HarmonyHelper and use as implementation for EmptyMethod?

Reply:
resolved